Frameworks/UIKit/UISegment.h

```diff
@@ -13,14 +13,16 @@
 // THE SOFTWARE.
 //
 //******************************************************************************
+#pragma once
 
 @interface UISegment : UIView {
 @public
     id _delegate;
     idretain _title;
-    idretain _image;
-    idretain _selectedBackground, _highlightedBackground;
-    idretain _dividerImage;
+    UIImage* _image;
+    UIImage* _selectedBackground;
+    UIImage* _highlightedBackground;
+    UIImage* _dividerImage;
```

Comment:
looks like, thanks!

---
In reply to: [95006851](https://github.com/Microsoft/WinObjC/pull/1643#discussion_r95006851) [](ancestors = 95006851)